keyboards/cocoa40/keymaps/default/keymap.c

```diff
@@ -0,0 +1,53 @@
+/* Copyright 2019 'Naoto Takai'
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+
+#define _BASE 0
```

Comment:
Instead of defining each layer to a specific index, we recommend using an enum to handle the layer indexes.

Replacing Lines 19-22 with:

```c
enum layer_names {
	_BASE,
	_NUMS,
	_SYMS,
	_FN,
};
```

... is programmatically identical to what you have now, but changing the layers only involves rearranging the values in the enum.